### ISSUE TITLE
TypeError on the README example

### ISSUE THREAD

**Issue by edcrypt** [eevee/camel#3]
Trying out by %cpaste-ing the code into ptipython, rev 9e2720b
```
TypeError                                 Traceback (most recent call last)
<ipython-input-2-57b1790be047> in <module>()
     21 value = DieRoll(3, 6)
     22 camel = Camel([reg])
---> 23 print(camel.dump(value))
     24 
     25 # !roll 3d6

/home/edcrypt/dev/camel/camel/__init__.py in dump(self, data)
    113         dumper = self.make_dumper(stream)
    114         dumper.open()
--> 115         dumper.represent(data)
    116         dumper.close()
    117         return stream.getvalue()

/usr/lib/python2.7/dist-packages/yaml/representer.pyc in represent(self, data)
     26 
     27     def represent(self, data):
---> 28         node = self.represent_data(data)
     29         self.serialize(node)
     30         self.represented_objects = {}

/usr/lib/python2.7/dist-packages/yaml/representer.pyc in represent_data(self, data)
     55             data_types = self.get_classobj_bases(data.__class__)+list(data_types)
     56         if data_types[0] in self.yaml_representers:
---> 57             node = self.yaml_representers[data_types[0]](self, data)
     58         else:
     59             for data_type in data_types:

/home/edcrypt/dev/camel/camel/__init__.py in run_representer(self, representer, tag, dumper, data)
    214                 "Representers must return native YAML types, but the representer "
    215                 "for {!r} returned {!r}, which is of type {!r}"
--> 216                 .format(data, canon_value, canon_type))
    217 
    218     def inject_dumpers(self, dumper, version_locks=None):

TypeError: Representers must return native YAML types, but the representer for DieRoll(3,6) returned '3d6', which is of type <type 'str'>
```
Python 2.7.10 on Ubuntu 15.10.

**Comment by eevee** [comment]
Ah, whoops, that needs to be a `unicode` in Python 2...

**Comment by eevee** [comment]
Fixed by 1a0d1b4, thanks!
